packages/theming/win32-theme/src/NativeModule/getThemingModule.native.ts

```diff
@@ -19,7 +22,7 @@ function disableGetPalette(): boolean {
 }
 
 export function getThemingModule(): [OfficeThemingModule, NativeEventEmitter | undefined] {
-  const module = NativeModules && NativeModules.Theming;
+  const module = TurboModuleRegistry.get('Theming');
```

Comment:
Would this be backwards compatible with Native Modules, or does this only work for TurboModules?

Reply:
https://github.com/facebook/react-native/blob/d8d9c28b482ed294770592ecb75bd6cc7f5c9da7/Libraries/TurboModule/TurboModuleRegistry.js#L21

Yes - it'll try to get a NativeModule if it exists for backwards compat.